iOS/MobileStarter/react-app/src/IModelDownloader.tsx

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { useIsMountedRef } from "@itwin/mobile-ui-react";
+import { Project } from "@itwin/projects-client";
+import { ProgressInfo } from "@itwin/core-frontend/lib/cjs/request/Request";
+import { DownloadBriefcaseOptions, NativeApp } from "@itwin/core-frontend";
+import { MinimalIModel } from "@itwin/imodels-client-management";
+import { BentleyError, BriefcaseDownloader, BriefcaseStatus, IModelStatus, SyncMode } from "@itwin/core-common";
+import { ProgressRadial } from "@itwin/itwinui-react";
+import { Button, i18n, presentError, IModelInfo } from "./Exports";
+
+async function downloadIModel(project: Project, iModel: MinimalIModel, handleProgress: (progress: ProgressInfo) => boolean): Promise<IModelInfo> {
+  const opts: DownloadBriefcaseOptions = { syncMode: SyncMode.PullOnly };
+  let downloader: BriefcaseDownloader | undefined;
+  let canceled = false;
+  try {
+    downloader = await NativeApp.requestDownloadBriefcase(project.id, iModel.id, opts, undefined, (progress: ProgressInfo) => {
+      if (!handleProgress(progress)) {
+        console.log("Canceling download.");
```

Comment:
While this is probably left over from my code, it should probably be removed.

Reply:
Removed